tests/PHPStan/Rules/Methods/data/returnTypes.php

```diff
@@ -349,7 +349,7 @@ public function returnsNullInTernary(): int
 		}
 	}
 
-	public function misleadingBoolReturnType(): boolean
+	public function misleadingBoolReturnType(): bool
```

Comment:
I don't want these changes, they're here on purpose

Reply:
I apologise, restored, thank you.